src/main/java/com/bbd/modisa/client/BankApp.java

```diff
@@ -42,16 +38,15 @@ private static void chequeSavings(AccountService accountServices)
                 System.out.print("Your Current Balance is = R " + String.format("%.2f",  accountServices.getBalance()));
             } else if (optS == 'L') {
                 AccountDB.getAllTransactions(accountNo);
-                log();
+                log(new SavingsAccountService());
```

Comment:
why are you creating a new object here? we already have an acccount service that you receive as a parameter